assets/js/modules/reader-revenue-manager/datastore/publications.test.js

```diff
@@ -150,107 +141,7 @@ describe( 'modules/reader-revenue-manager publications', () => {
-				const publication = fixtures.publications[ 3 ];
-
-				// Set the current settings.
-				const settings = {
-					publicationID: publication.publicationId,
-					publicationOnboardingState:
-						PUBLICATION_ONBOARDING_STATES.UNSPECIFIED,
-				};
-
-				fetchMock.getOnce( publicationsEndpoint, {
-					body: fixtures.publications,
-					status: 200,
-				} );
-
-				fetchMock.postOnce( settingsEndpoint, {
-					body: {
-						...settings,
-						publicationOnboardingState:
-							PUBLICATION_ONBOARDING_STATES.ONBOARDING_COMPLETE,
-					},
-					status: 200,
-				} );
-
-				registry
-					.dispatch( MODULES_READER_REVENUE_MANAGER )
-					.receiveGetSettings( settings );
-
-				expect(
-					registry
-						.select( CORE_UI )
-						.getValue(
-							UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION
-						)
-				).toBeUndefined();
-
-				await registry
-					.dispatch( MODULES_READER_REVENUE_MANAGER )
-					.syncPublicationOnboardingState();
-
-				// Ensure that the UI key is set to true.
-				expect(
-					registry
-						.select( CORE_UI )
-						.getValue(
-							UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION
-						)
-				).toBe( true );
-			} );
-
-			it( 'should not set UI_KEY_SHOW_RRM_PUBLICATION_APPROVED_NOTIFICATION to true in CORE_UI store if onboarding state does not change', async () => {
-				const publication = fixtures.publications[ 3 ];
-
-				// Set the current settings.
-				const settings = {
-					publicationID: publication.publicationId,
-					publicationOnboardingState:
-						PUBLICATION_ONBOARDING_STATES.ONBOARDING_COMPLETE,
-				};
-
-				fetchMock.getOnce( publicationsEndpoint, {
-					body: fixtures.publications,
-					status: 200,
-				} );
-
-				fetchMock.postOnce( settingsEndpoint, {
-					body: {
-						...settings,
-						publicationOnboardingState:
-							PUBLICATION_ONBOARDING_STATES.ONBOARDING_COMPLETE,
-					},
-					status: 200,
-				} );
-
-				registry
-					.dispatch( MODULES_READER_REVENUE_MANAGER )
-					.receiveGetSettings( settings );
-
-				expect(
-					registry
-						.select( CORE_UI )
-						.getValue(
-							UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION
-						)
-				).toBeUndefined();
-
-				await registry
-					.dispatch( MODULES_READER_REVENUE_MANAGER )
-					.syncPublicationOnboardingState();
-
-				// Ensure that the UI key is set to true.
-				expect(
-					registry
-						.select( CORE_UI )
-						.getValue(
-							UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION
-						)
-				).toBeUndefined();
-			} );
-
-			it( 'should not update the timestamp, call the saveSettings endpoint, and set UI_KEY_SHOW_RRM_PUBLICATION_APPROVED_NOTIFICATION to true in CORE_UI store if there is no saved publication', async () => {
+			it( 'should not update the timestamp if onboarding state has not changed', async () => {
```

Comment:
This test can be removed as there's no longer any timestamp specific logic in the `syncPublicationOnboardingState()` action.